src/commands/addSshKey.ts

```diff
@@ -16,7 +16,7 @@ import { configureSshConfig, sshFsPath } from "../utils/sshUtils";
 
 export async function addSshKey(context: IActionContext, node?: VirtualMachineTreeItem): Promise<void> {
     if (!node) {
-        node = await ext.tree.showTreeItemPicker<VirtualMachineTreeItem>(VirtualMachineTreeItem.linuxContextValue, context);
+        node = await ext.tree.showTreeItemPicker<VirtualMachineTreeItem>(/^VirtualMachine.+linux.+running/, context);
```

Comment:
I think adding a context value constant for this would be useful, especially since its used in two places. Also couldn't you use the string "VirtualMachine-linux-running" here since it's an exact match?